src/networkDetails.ts

```diff
@@ -45,9 +45,9 @@ export const addresses: IAddresses = {
     TREASURY_ADDRESS: "0x0d722D813601E48b7DAcb2DF9bae282cFd98c6E7",
     REDEEM_HELPER_ADDRESS: "0xBd35d8b2FDc2b720842DB372f5E419d39B24781f",
     PT_TOKEN_ADDRESS: "0x0a2d026bacc573a8b5a2b049f956bdf8e5256cfd", // 33T token address, taken from `ticket` function on PRIZE_STRATEGY_ADDRESS
-    PT_PRIZE_POOL_ADDRESS: "0xf9081132864ed5e4980CFae83bDB122d86619281",
-    PT_PRIZE_STRATEGY_ADDRESS: "0x2Df17EA8D6B68Ec444c9a698315AfB36425dac8b",
-    GIVING_ADDRESS: "0x83D4FE6Ead62547758E094ee5BDb343ADbC2AeB9",
+    PT_PRIZE_POOL_ADDRESS: "0xf9081132864ed5e4980CFae83bDB122d86619281", // NEW
+    PT_PRIZE_STRATEGY_ADDRESS: "0x2Df17EA8D6B68Ec444c9a698315AfB36425dac8b", // NEW
+    GIVING_ADDRESS: "0x5Bb351eeC15A703Afe368f9aa49ccC7de45e5a21",
```

Comment:
@appleseed-iii  - do these addresses need to be managed in two places? I know you had done some refactor work here.

Reply:
resolved in https://github.com/OlympusDAO/olympus-frontend/pull/1550/commits/7288cb9bd85b34f646f021cc44796a9ee7fae6d2